Bio/SeqRef/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 by Tianyi Shi.  All rights reserved.
```

Comment:
We should perhaps not call this ``Bio.SeqRef`` and pick something lower case (PEP8 guidelines)

Reply:
Actually do we need a new top-level module for this?

Reply:
Maybe not, but where would you suggest putting it?

Reply:
I guess it depends where `SeqRef` will be used. @TianyiShi2001  If you could add some documentation to the tutorial, then we can have a better idea of how this will be used in practice.